README.md

```diff
@@ -3,39 +3,65 @@
 ## Installation
 1. Clone repository into workspace [It doesn't have to be called moveit_simple_ws]
 ```
-mkdir moveit_simple_ws && cd moveit_simple_ws
-mkdir src && cd src
-catkin_init_workspace
+mkdir -p moveit_simple_ws/src && cd moveit_simple_ws/src
 git clone -b kinetic-devel https://github.com/plusone-robotics/moveit_simple.git
 ```
 
-2. Source Dependencies
+2. Get Source Dependencies
 ```
+cd moveit_simple_ws/src
```

Comment:
You should already be in moveit_simple_ws/src from the first [command](https://github.com/plusone-robotics/moveit_simple/pull/84/files#diff-04c6e90faac2675aa89e2176d2eec7d8R6). This gives a "No such file or directory" error as written.

Reply:
good point. Fixed